docs/src/api/utility_api/parameter/parameter_manager.rst

```diff
@@ -0,0 +1,198 @@
+Parameter Manager
+==================
+
+Overview
+--------
+
+The ``ParameterManager`` inherits from the ActionManager.It provides a user interface for managing hierarchical parameter configurations with search, save/load capabilities, and a collapsible toolbar.
+
+Quick Start
+-----------
```

Comment:
should say it's a layer on top of the Parameter and ParameterTree helping classes to handle changes in the settings and providing a unique interface for all modules in pymodaq dealing with Parameters